src/Nethermind/Nethermind.Provide.Vault/VaultTxSender.cs

```diff
@@ -36,7 +36,7 @@ public class VaultTxSender : ITxSender
             {1, new Guid("deca2436-21ba-4ff5-b225-ad1b0b2f5c59")},
             {3, new Guid("66d44f30-9092-4182-a3c4-bc02736d6ae5")},
             {4, new Guid("07102258-5e49-480e-86af-6d0c3260827d")},
-            {5, new Guid("1b16996e-3595-4985-816c-043345d22f")},
+            //{5, new Guid("1b16996e-3595-4985-816c-043345d22f")},
```

Comment:
The GUID here was incorrect, because of that, Vault plugin loading was failing. Should I change to any correct GUID, or it has some special meaning?

Reply:
check with Eoin